batch/batch/driver/main.py

```diff
@@ -87,6 +87,13 @@ def instance_from_request(request):
     return inst_coll_manager.get_instance(instance_name)
 
 
+# Old workers use the Authorization header for their identity token
```

Comment:
After old workers die, we can remove this code, right?

Reply:
Ya